x-pack/plugins/apm/public/components/app/Settings/agent_configurations/List/index.tsx

```diff
@@ -245,7 +245,7 @@ export function AgentConfigurationList({
         items={configurations}
         initialSortField="service.name"
         initialSortDirection="asc"
-        initialPageSize={20}
+        hidePerPageOptions={false}
```

Comment:
is `true` a sensible default here for ManagedTable? from the issue description it looks like we want to show per page options, with some exceptions, rather than the other way around?